event-log/src/main/scala/io/renku/eventlog/Microservice.scala

```diff
@@ -36,7 +36,7 @@ import io.renku.eventlog.latestevents.IOLatestEventsEndpoint
 import io.renku.eventlog.metrics._
 import io.renku.eventlog.processingstatus.IOProcessingStatusEndpoint
 import io.renku.eventlog.statuschange.IOStatusChangeEndpoint
-import io.renku.eventlog.subscriptions.{EventsDispatcher, IOSubscriptionsEndpoint, Subscriptions}
+import io.renku.eventlog.subscriptions.{EventsDistributor, IOSubscriptionsEndpoint, Subscriptions}
```

Comment:
Do we also want to rename `Subscriptions` class to `Subscribers` (or smth better)?

Reply:
Yes correct!